src/actions/transactions.js

```diff
@@ -188,8 +188,8 @@ export const transactionsUpdated = ({
           type: actionTypes.transactionsUpdated,
         });
         // eslint-disable-next-line no-constant-condition
-        if (pendingTransactions.length || true) {
-          // "|| true" above was added to disable this, because this caused pending transactions
+        if (pendingTransactions.length && false) {
```

Comment:
why not commenting the whole section and moving the TODO at the top of the comment and linking the core 1.0.0 issue?